assets/pages/new-thank-you/thankYou.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { init as pageInit } from 'helpers/page/page';
+import { renderPage } from 'helpers/render';
+import { detect, countryGroups, type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+import Page from 'components/page/page';
+import Footer from 'components/footer/footer';
+import SvgArrowRight from 'components/svgs/arrowRightStraight';
+import { NewHeader } from 'components/headers/new-header/Header';
+import { createPageReducerFor } from '../contributions-landing/contributionsLandingReducer';
+
+
+// ----- Redux Store ----- //
+
+const countryGroupId: CountryGroupId = detect();
+
+const store = pageInit(createPageReducerFor(countryGroupId));
+
+const reactElementId = `new-thank-you-page-${countryGroups[countryGroupId].supportInternationalisationId}`;
```

Comment:
Maybe `new-contributions-thank-you-page`? We have thank you pages for subs as well.

Reply:
👍